doc/Config.md

```diff
@@ -1,8 +1,198 @@
-Config values that can only be set in .htconfig.php
-===================================================
+Friendica Configuration
```

Comment:
The title was chosen, as there is already a help file for "Settings" dealing with the admin panel and this file is for documentation of the *hidden* config variables.

Reply:
I know, but I needed a place to detail the migration steps. Please advise how I should proceed.

Reply:
You can still add the section on how to migrate from the old format to the new one at the bottom of the documentation file about the *Config values that can only be set in config/...* (what ever the filename is then).

Or you add that section to the documentation file about how to update in general.